### PR TITLE
Update community card

### PR DESCRIPTION
**Pull request by peteclodi**
## Proposed changes

Update the community info data styling so that the 2 columns are lined up properly
Remove the sizing and location setting for the dropdown menu
Move the %selection-base placeholder file from PBA to common

## Types of changes

_What types of changes does your code introduce? Put an `x` in all the boxes that apply:_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Unit Tests (only unit tests were added)
- [ ] Sprint Deployment

## How has this been tested?

Manual UI Testing in PBQ and PBA

## Checklist

_*Pull Requester:* Put an `x` in the boxes that apply. If not applicable, leave the checkbox blank. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask the reviewer._

_*Reviewer:* If you see a blank checkbox where you  believe the action is applicable to the PR you are reviewing (and therefore should have an `x`), engage in a conversation (verbal or over Github comments) with the requester to understand their logic._

- [x] I have added the current sprint milestone to this PR.
- [x] I have attached this PR to all relevant cards.
- [ ] Any dependent changes have been merged and released if needed.
- [x] My code follows the code style of this project.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have added tests (unit and/or acceptance) that prove my fix is effective or that my feature works.
- [ ] My changes have manual deployment steps, and I have documented these in the current sprint's deployment card.
- [ ] I have checked off and verified that all items within the “Definition of Done” section on this work item have been met.

## Potential opportunities or challenges uncovered as a result of this change

- [ ] I have created inbox cards for any opportunities or future challenges that I was able to recognize in the process making this change.

## Further comments 

_Any other notes you wish to add. If this is a relatively large or complex change, it is recommended to explain why you chose the solution you did and what alternatives you considered, etc (though this can also be conveyed in-person)._

### CODE REVIEW

**Comment by peteclodi** [comment]
[(3) A professor/school admin user may click a button in their my communities dashboard to create a new community [3]](https://trello.com/c/xVOx4q2h/1904-(3)-a-professor%2Fschool-admin-user-may-click-a-button-in-their-my-communities-dashboard-to-create-a-new-community-%5B3%5D)

File: assets/scss/placeholders/_selection-base.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                background-color: $admin-secondary;
+            }
+        }
+    }
+
+    &>.editable {
+        &.pb-autocomplete {
+            &>input {
+                &:focus + ul.f-dropdown {
+                    box-shadow: 0 0 5px $admin-primary_light;
```
Comment (rionner) [inline assets/scss/placeholders/_selection-base.scss]: Use the mixin `@include box-shadow-focus($primary-color_light, $primary-color_dark)` for consistency - you will also be able to remove the `border-color` declaration on line 42

File: assets/scss/placeholders/_selection-base.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        padding: 0;
+
+        &>.name {
+            @extend input;
+            white-space: nowrap;
+            overflow-y: hidden;
+        }
+
+        &>.entypo-cancel {
+            background-color: $PB_grey;
```
Comment (rionner) [inline assets/scss/placeholders/_selection-base.scss]: Update the color variables in this document with the variables defined here: http://dev.style-guide.packback.co/atoms#colors - wherever an "admin" specific color variable is used, replace it with the equivalent generic color (ie: $primary-color)

File: assets/scss/placeholders/_selection-base.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            }
+            .f-dropdown {
+                @extend %typeahead-dropdown;
+                @extend %custom-scrollbar;
+                position: inherit;
+                max-width: 100%;
+                margin-top: rem-calc(-3);
+                border-top-left-radius: 0;
+                border-top-right-radius: 0;
+                box-shadow: 0 0 5px $admin-primary_light;
```
Comment (rionner) [inline assets/scss/placeholders/_selection-base.scss]: Use the mixin @include box-shadow-focus($primary-color_light, $primary-color_light) for consistency - you will also be able to remove the border-color declaration on line 42

File: assets/scss/organisms/_community-card.scss [hunk 4]
```diff
@@ -16,6 +16,32 @@
     .info {
         @include pb-panel($secondary-background, $primary-background);
         width: 100%;
+
```
Comment (peteclodi) [inline assets/scss/organisms/_community-card.scss]: I'm making a best guess based upon how the change section should've always worked, except that it's visibility functionality was broken for a long time.
My feeling is that we should keep this as it is and confirm with Jess on how we want to deal with the Change Section functionality going forward.
If you feel that it is a problem in the look & feel then I can comment out the html code until we can revisit it with Jess.
Let me know what route you'd prefer.

**Comment by peteclodi** [comment]
@rionner Updated based upon your feedback, but kept the `actions` class in the community card since it allowed for dynamic sizing of the dropdown menu.

File: assets/scss/organisms/_community-card.scss (hunk 4 above)
Comment (rionner) [inline assets/scss/organisms/_community-card.scss]: @peteclodi -  I think you're all good, just wasn't sure if there was a comp I should see before "approving"

**Pull request merged**